code/modules/projectiles/guns/misc/syringe_gun.dm

```diff
@@ -33,7 +36,7 @@
 /obj/item/gun/syringe/process_chamber()
 	if(chambered && !chambered.BB) //we just fired
 		recharge_newshot()
-
+	update_icon()
```

Comment:
add a new line after this between the new proc

Reply:
alrighty I added it